### PR TITLE
VACMS-7245 test dependency updates & vagovclaro test fixes

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #7245 & #7914. hat tip to @ndouglas for the help

In order to switch to the new theme for all users by default, existing tests need to pass with that theme so that builds do not fail everywhere. In an effort to improve tests, I bumped up `cypress` and related testing dependencies. This also solves the two `high` level warnings in npm audit.

Cypress is now pinned to `8.5.0` in order to run in docker locally & GHA without crashing. See this github issue: https://github.com/cypress-io/cypress/issues/18681

I changed two selectors to use the input's `#id` instead of the `.field-foo-class input` because the ID is the same in both themes, but the class prints differently (claro uses BEM by default so it prints `.field--foo-class` instead, causing errors).

## Testing done
locally `ddev test va/tests/accessibility` and `ddev test va/tests/behavioral` with vagovadmin as the active + default theme, and then again with vagovclaro as the active+default theme.

## Screenshots

**with vagovadmin enabled as the admin & default theme:**
`ddev test va/tests/accessibility`
```

       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  homepageAccessibility.spec.js            00:02        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  nodeCreationAccessibility.spec.js        09:26       16       16        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        09:28       17       17        -        -        -
```

`ddev test va/tests/behavioral`
```
       Spec                                              Tests  Passing  Failing  Pending  Skipped
┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  content_release.feature                  01:16        5        4        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  entity_reference_validation.feature      00:54        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  google_tag_manager.feature               03:06       11       11        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  help_center.feature                      00:31        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  image_crop.feature                       609ms        2        -        -        2        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  password_policy.feature                  01:52        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  text_validation.feature                  01:21        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  tooltip.feature                          00:08        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_login.feature                       00:18        1        1        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        09:30       28       25        -        3        -
```

**with vagovclaro enabled as the admin & default theme:**
`ddev test va/tests/accessibility`

```
       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  homepageAccessibility.spec.js            00:02        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  nodeCreationAccessibility.spec.js        10:01       16       16        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        10:03       17       17        -        -        -

```
`ddev test va/tests/behavioral`
```
      Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  content_release.feature                  01:17        5        4        -        1        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  entity_reference_validation.feature      00:52        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  google_tag_manager.feature               03:01       11       11        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  help_center.feature                      00:32        3        3        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  image_crop.feature                       561ms        2        -        -        2        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  password_policy.feature                  01:50        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  text_validation.feature                  01:17        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  tooltip.feature                          00:08        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  user_login.feature                       00:19        1        1        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        09:20       28       25        -        3        -

```

## QA steps
pull branch, run test suite with both themes and see no failures

### Definition of Done
- [x] Documentation has been updated, if applicable.
- [x] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [x] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [x] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?
- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner
- [ ] `Needs PO review`

### CMS user-facing announcement
Is an announcement needed to let editors know of this change?
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/themes/custom/vagovclaro/assets/js/anchor_links.es6.js [hunk 1]
```diff
@@ -25,17 +25,20 @@
    */
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/assets/js/anchor_links.es6.js:25]: noting this code matches `vagovadmin/assets/js/anchor_links.es6.js`, that file was updated in november after i originally copied things over. no new logic here

**Comment by tjheffner** [comment]
This is with `vagovadmin` enabled on this environment.
Screen Shot 2022-02-10 at 8 41 33 AM

I'm going to merge main in and switch to vagovclaro on the next deployment, see that tests pass there as well.

**Comment by swirtSJW** [comment]
@tjheffner Just double checking, is this still a WIP? or is it ready for me to look at?

**Comment by tjheffner** [comment]
All tests have passed in vagovclaro!
Screen Shot 2022-02-10 at 8 41 33 AM

@swirtSJW it's no longer WIP, but I wanted to make sure this round of tests passed in tugboat before I changed the title :)

**Review by swirtSJW — APPROVED** [review]
Nice work.  And thanks for the extra rounds of testing with both admin themes running.

**Pull request merged**